src/databricks/labs/ucx/source_code/known-readme.txt

```diff
@@ -0,0 +1,12 @@
+Known libraries that are not in known.json
```

Comment:
move that to `known.py` file comment

Reply:
updating comments in a separate PR just now, but not sure what you mean by file comment (JSON does not support comments)?